ligo/haskell/test/Test/Ligo/RegistryDAO.hs

```diff
@@ -191,6 +191,51 @@ test_RegistryDAO =
               -- Now we expect this to work
               withSender (AddressResolved wallet1) $
                 call baseDao (Call @"Propose") (ProposeParams requiredFrozen largeProposalMeta)
+
+    , nettestScenarioCaps "checks on-chain view correctly returns the registry value" $ do
+        let frozen_scale_value = 1
+        let frozen_extra_value = 0
+        let slash_scale_value = 1
+        let slash_division_value = 1
+        withOriginated 3
+          (\(admin: wallet1: voter1:_) -> setExtra @Natural [mt|slash_division_value|] 2 $
+            setExtra @Natural [mt|frozen_scale_value|] frozen_scale_value $
+            setExtra @Natural [mt|frozen_extra_value|] frozen_extra_value $
+            setExtra @Natural [mt|slash_scale_value|] slash_scale_value $
+            setExtra @Natural [mt|max_proposal_size|] 200 $
+            setExtra @Natural [mt|slash_division_value|] slash_division_value $ initialStorageWithExplictRegistryDAOConfig admin [wallet1, voter1]) $
```

Comment:
`initialStorageWithExplictRegistryDAOConfig` already sets the `frozen_scale_value`, `frozen_extra_value`, `slash_scale_value` and `slash_division_value` to the values you set up above, so I think only setting `max_proposal_size` should be enough. Also, you're setting `slash_division_value` twice.

Reply:
Makes sense. I think I wanted to make the values being used explicit. But since they just result in the proposal being rejected or not, I think it is not a big deal, so I have made the changes. Have left some comments though.